packages/core/contracts/data-verification-mechanism/implementation/VotingV2.sol

```diff
@@ -517,7 +456,7 @@ contract VotingV2 is Staker, OracleInterface, OracleAncillaryInterface, OracleGo
         bytes memory ancillaryData,
         int256 salt
     ) public override nonReentrant() onlyIfNotMigrated() {
```

Comment:
Nit: all the modifiers that don't have arguments can be use without parentheses:
`) public override nonReentrant onlyIfNotMigrated {`

Reply:
let's leave that for a follow on PR I think.